src/Reports/SiteSummary.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class SiteSummary extends SS_Report
```

Comment:
Can you please add a class doc block? Also try and use namespaces wherever you can, I think that's largely anything that isn't a DataObject at the moment